tests/mockServer/dateSynchronization.js

```diff
@@ -0,0 +1,105 @@
+const fs = require('fs')
+
+// format data in accordance with ISO 8601 with ms
+function formatDate(date) {
+    return date.toISOString()
+}
+
+function updateRuns() {
+    // Current time
+    const now = new Date()
+
+    // Past hour
+    const oneHourAgo = new Date(now.getTime() - (45 * 60 * 1000))
+
+    // Past 24h
+    const oneDayAgo = new Date(now.getTime() - (20 * 60 * 60 * 1000))
+
+    fs.readFile('./tests/mockServer/data/runs.json', 'utf8', (err, data) => {
+        if (err) {
+            console.error('Error reading the file:', err)
+            return
+        }
+
+        let runsData
+        try {
+            runsData = JSON.parse(data)
+        } catch (parseErr) {
+            console.error('Error parsing JSON:', parseErr)
+            return
+        }
+
+        // update start_time та last_update
+        const updatedRuns = runsData.runs.map(run => {
```

Comment:
You can use such condition instead of many if-else (except the last one with 'oneHourAgo'), I am not sure if we need to check name if we check the uid
`if (['cf842616c89347c7bb7bca2c9e840a21', '76f48c8165da473bb4356ef7b196343f' ,'...' ].includes(run.metadata.uid)) {
  return {
                    ...run,
                    status: {
                        ...run.status,
                        start_time: formatDate(oneDayAgo),
                        last_update: formatDate(oneDayAgo)
                    }
                }
}`